prime-router/metadata/fhir_mapping/hl7/datatypes/CX/Identifier.yml

```diff
@@ -0,0 +1,190 @@
+resourceType: Identifier
+
+value:
+  type: STRING
+  valueOf: CX.1
+  expressionType: HL7Spec
+
+extension_1:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: identifier-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.2
+
+extension_2:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: naming-system-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.3
+
+# Checking if HD is an Organization or a System by looking at the ID type.
+# If ID type is null or ID type is not ISO or UUID, treat as organization.
+# Otherwise, treat as system.
+assigner_1:
+  vars:
+    cx43: String, CX.4.3
+  condition: cx43 NULL
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+assigner_2:
+  vars:
+    cx43: String, CX.4.3
+  condition: $cx43 NOT_NULL && $cx43 NOT_EQUALS ISO && $cx43 NOT_EQUALS UUID
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+system:
+  vars:
+    cx41: String, CX.4.1
+    cx43: String, CX.4.3
+  expressionType: nested
+  condition: $cx43 EQUALS ISO || cx43 EQUALS UUID
+  expressions:
+    - condition: $cx41 NOT_NULL
+      expressionType: HL7Spec
+      type: STRING
+      valueOf: CX.4.1
+    - condition: $cx41 NULL && cx43 EQUALS ISO
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:oid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+    - condition: $cx41 NULL && cx43 EQUALS UUID
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:uuid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+
+extension_system:
```

Comment:
What is this one used for? Not seeing it used in the FHIR->HL7 mapping, and not seeing it in the HL7->FHIR mapping inventory.

Reply:
Is is line 9 in the CX spreadsheet: https://docs.google.com/spreadsheets/d/1wZsWZMw0kBpu-oWB1U3iljL0m_7PLGM9tFfSqBUoRU4/edit?usp=sharing

Reply:
Line 9 in the CX spreadsheet is the mapping to `system`, but this is actually just putting an extension onto the top-level `identifier`. Some kind of extension on the identifier might be necessary to preserve HL7->FHIR->HL7, but if it is, it should have a more specific url, also it wouldn't be able to be handled as a simple string, since CX.4 is an HD which can have multiple fields. This will likely need to be resolved for the tests to work after updating `cx-identifier-system.hl7` to utilize the `system` case instead of `assigner`.